src/matrixrtc/EncryptionManager.ts

```diff
@@ -80,8 +76,11 @@ export class EncryptionManager implements IEncryptionManager {
     // if it looks like a membership has been updated.
     private lastMembershipFingerprints: Set<string> | undefined;
 
-    private currentEncryptionKeyIndex = -1;
-
+    // TODO: remove this value since I think this is not helpful to use
+    // it represents what index we actually sent over to ElementCall (which we do in a delayed manner)
+    // but it has no releavnt information for the encryption manager.
+    private mediaTrailerKeyIndexInUse = -1;
+    private latestGeneratedKeyIndex = -1;
```

Comment:
Not sure why you're adding something and then claiming in a comment that it should be removed...

Also, why the rename?

Reply:
Mostly because of: https://github.com/matrix-org/matrix-js-sdk/pull/4785#discussion_r2035046112

But you might be right. we did the testing where we needed the logging so probably its best to just remove it.